### ISSUE TITLE
Animation flickering

### ISSUE THREAD

**Issue by edgar-zigis** [Nightonke/VHBoomMenuButton#12]
Hello,

while using this menu I encountered a strange animation flickering. For example when you open menu and then decide to close, you can see flickering button layers on top of the "pieces" (for example using CircleMenu).

**Comment by qaavi** [comment]
Hi,

Any updates on above issue ?

**Comment by AhmedShammala** [comment]
+1

**Comment by AhmedShammala** [comment]
+2

**Comment by yasirmturk** [comment]
+3

**Comment by qaavi** [comment]
This issue is related to Material design ( https://github.com/CosmicMind/Material ). When you use "Material design Navigation Controller" and this VHBoomMenuButton together, this issue occurs.

Used UINavigationController and this issue disappeared.

**Comment by yasirmturk** [comment]
@AhmedShammala @edgar-zigis are you also using [Material](https://github.com/CosmicMind/Material) ?

**Comment by edgar-zigis** [comment]
No, I don't use it on any Navigation Controller and I don't use Material library. 
I simply put it inside the view of simple UIViewController and it has no specific options or extensions which could cause side effects. Tried both using autolayout and simple rect frames - same result.

**Comment by febinfathah** [comment]
I am also having the same issue of Flickering animation.

**Comment by AhmedShammala** [comment]
@yasirmturk  no but it was bz i using "CRToast" and fix after change it .
